Documentation/PageTsconfig/Mod/WebLayout.rst

```diff
@@ -1,327 +1,350 @@
-   allowing you to create independent content elements in a given language.
+Example: Allow inconsistent language modes
+------------------------------------------
 
-   If you want to go back to the old, inconsistent behavior, you can toggle it back on using this switch.
+Allows to set TYPO3s page module back to inconsistent language mode
 
-:aspect:`Example`
-   Allows to set TYPO3s page module back to inconsistent language mode
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
 
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+    mod.web_layout.allowInconsistentLanguageHandling = 1
 
-      mod.web_layout.allowInconsistentLanguageHandling = 1
 
+..  index:: BackendLayouts
 
-.. index:: BackendLayouts
+..  _mod-web-layout-backendLayouts:
 
 BackendLayouts
 ==============
 
-:aspect:`Datatype`
-   array
+..  confval:: BackendLayouts
+    :name: mod-web-layout-BackendLayouts
+    :type: array
 
-:aspect:`Description`
-   Allows to define backend layouts via Page TSconfig directly, without using database records.
+    Allows to define backend layouts via Page TSconfig directly, without using database records.
 
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+..  _mod-web-layout-backendLayouts-example:
 
-      mod.web_layout.BackendLayouts {
-         exampleKey {
+Example: Define a backend layout
+--------------------------------
+
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
+
+    mod.web_layout.BackendLayouts {
+        exampleKey {
             title = Example
             icon = EXT:example_extension/Resources/Public/Images/BackendLayouts/default.gif
             config {
-               backend_layout {
-                  colCount = 1
-                  rowCount = 2
-                  rows {
-                     1 {
-                        columns {
-                           1 {
-                              name = LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:colPos.I.3
-                              colPos = 3
-                              colspan = 1
-                           }
+                backend_layout {
+                    colCount = 1
+                    rowCount = 2
+                    rows {
+                        1 {
+                            columns {
+                                1 {
+                                    name = LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:colPos.I.3
+                                    colPos = 3
+                                    colspan = 1
+                                }
+                            }
                         }
+                        2 {
+                            columns {
+                                1 {
+                                    name = Main
+                                                colPos = 0
```

Comment:
indentation is here wrong. Also closing brackets later.

Reply:
Indentation of large files inline rst is hard to handle. Moved the code example to an include file instead.